pages/common/webpack.md

```diff
@@ -0,0 +1,19 @@
+# webpack
+
+> Bundle a web project's js files and other assets into a single output file.
+
+- Create a single output file from an entry point file:
+
+`webpack {{app.js}} {{bundle.js}}`
+
+- Load css files too from your js file. (This uses the css loader for .css files):
```

Comment:
"your" --> "the". Also, remove the period before the parenthesis (since the whole sentence is punctuated with a colon in the end), and lowercase the word "this" inside the parenthesis,